hydragnn/preprocess/serialized_dataset_loader.py

```diff
@@ -186,9 +187,13 @@ def update_predicted_values(type: list, index: list, data: Data):
     """
     output_feature = []
     data.y_loc = torch.zeros(1, len(type) + 1, dtype=torch.int64, device=data.y.device)
+    index_counter_y = 0
     for item in range(len(type)):
         if type[item] == "graph":
-            feat_ = torch.reshape(data.y[index[item]], (1, 1))
+            feat_ = torch.reshape(
+                data.y[index_counter_y : output_dim[item]], (output_dim[item], 1)
```

Comment:
It should be something like below?
```suggestion
                data.y[index_counter_y : index_counter_y + output_dim[item]], (output_dim[item], 1)
```